docs/journal.md

```diff
@@ -106,6 +106,18 @@ pageClass: routes
 
 </Route>
 
+### 最新文章
+
+<Route author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/date/:sortType?" :paramsDesc="['期刊名称， 复制 URL 中 punumber 部分','排序方式, 默认`vol-only-seq`，复制 URL 中 sortType 部分']" radar="1" rssbud="1">
```

Comment:
```suggestion
<Route author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/ieee/:journal/latest/date/:sortType?" :paramsDesc="['期刊名称，复制 URL 中 punumber 部分','排序方式，默认`vol-only-seq`，复制 URL 中 sortType 部分']" radar="1" rssbud="1">
```

Reply:
Thanks your attention, I have modified it.